fitnessllm_dataplatform/stream/strava/request_utils.py

```
@@ -0,0 +1,115 @@
+    client_secret: str,
+    code: str,
+    grant_type: str = "authorization_code",
+) -> dict:
+    """Retrieve strava access token."""
+    payload = {
+        "client_id": client_id,
+        "client_secret": client_secret,
+        "code": code,
+        "grant_type": grant_type,
+    }
+    response = requests.post(StravaURLs.AUTH_URL.value, data=payload)
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)